src/richie/apps/demo/management/commands/create_demo_site.py

```diff
@@ -33,10 +33,10 @@
     SINGLECOLUMN_CONTENT,
     SUBJECTS_INFO,
 )
-from ...helpers import create_categories, recursive_page_creation
+from ...helpers import create_categories
 from ...utils import pick_image
 
-logger = logging.getLogger("richie.commands.core.create_demo_site")
+logger = logging.getLogger("richie.demo.commands.create_demo_site")
```

Comment:
Maybe we should adopt a logger naming convention such as `richie.{{ purpose }}.xxx` and not follow the related file path, WDYT?

```suggestion
logger = logging.getLogger("richie.commands.demo.create_demo_site")
```

Reply:
I changed to `richie.commands.demo.create_demo_site` as discussed.